lib/src/plaform/open_file.dart

```diff
@@ -26,10 +26,13 @@ class OpenFile {
       if (Platform.isMacOS) {
         _result = mac.system(['open', '$filePath']);
       } else if (Platform.isLinux) {
+        var filePathLinux = Uri.file(filePath!);
```

Comment:
this is the real fix for the special chars and spaces. it does assume that the linux.system call supports file uri's (xdg-open definitely does). I assume @marcusgregory has tested this and can validate this. I'll also test this